lib/ddtrace/configuration/settings.rb

```diff
@@ -232,11 +373,20 @@ def runtime_metrics(options = nil)
         end
       end
 
+      # TODO: profiler related. Ask @ivoanjo what this is for.
+      # @return [String,nil]
+      # @public_api
       option :site do |o|
         o.default { ENV.fetch(Ext::Environment::ENV_SITE, nil) }
         o.lazy
       end
```

Comment:
This is paired with `DD_API_KEY` when talking to the backend -- it's the actual deployment (us, eu, ...) [like here](https://docs.datadoghq.com/agent/troubleshooting/site/). My suggestion is to mark it in the same way as we do the api key -- by stating it's for internal Datadog use only.

Reply:
Thank you for that link above! Super helpful!

Documented as such.